### DISCUSSION TITLE
it's working on Clarett+ pre 4 ?

### DISCUSSION THREAD

**Discussion by betray32** [geoffreybennett/alsa-scarlett-gui#124] [Q&A]
it should be similar to Scarlett, but actually works with Clarett ? awesome job, thanks!

**Comment by geoffreybennett** [comment]
Hi @betray32,

Yes, all models of Scarlett Gen 2, 3, 4, Clarett, and Vocaster interfaces are supported.

Regards,
Geoffrey.
